src/Views/reportview.html

```diff
@@ -285,6 +285,38 @@ <h3 class="reportChartHeader">Estimated Flows</h3>
                         </div><!--end sc-->
                     </div><!--end extension-->
 
+                <div class="well well-sm"  ng-if="vm.regionService.selectedRegion.Applications.indexOf('EquationWeighting') != -1 && vm.nssService.selectedStatisticsGroupList.name.indexOf('Peak-Flow Statistics') != -1">
+                    <div class="wim-alert-warning" ng-show="vm.nssService.equationWeightingDisclaimers == true">Only 3 estimation methods can be weighted; the 3 estimation methods with lowest SEP values were weighted. </div>
+                    <h3>Methods Weighting</h3>
+                    <div ng-if="vm.nssService.equationWeightingResults.length == 0">No method weighting results returned</div>
+                    <div ng-if="vm.nssService.equationWeightingResults.length != 0">
+                        <div ng-repeat="rr in vm.nssService.equationWeightingResults">
+                            <h4>{{rr.RR}}</h4>

```

Comment:
Should Regression Region be listed here, if the results are area-averaged?

Reply:
I guess I didn't do to keep it similar to the statistics section of the report, we just have it listed as Area-Averaged in that section.
![image](https://user-images.githubusercontent.com/59063181/165302990-378b6691-c33b-4dd6-bfe3-5e2d4a80a2e7.png)

Reply:
Turns out this is an issue related to timeouts for the regression region results and area-averaged results.